lv_conf.h

```diff
@@ -17,7 +17,9 @@
 #ifndef LV_CONF_H
 #define LV_CONF_H
 
+#ifndef __ASSEMBLY__
```

Comment:
The include can be removed from here as it's included in lv_conf_internal.h too.

Reply:
It's included in lv_conf_template.h. So that should be remove too?

Reply:
Removed, it's also included in lv_types.h inside lv_conf_internal.h. It sounds reasonable.